tests/e2e/conftest.py

```diff
@@ -0,0 +1,84 @@
+# conftest.py
+"""
+Init file for tests.
+"""
+import logging
+import time
+
+import pytest
+from pymongo.errors import ServerSelectionTimeoutError
+
+from forum.mongo import get_database
+from forum.search.backend import ElasticsearchBackend
+from test_utils.client import APIClient
+
+log = logging.getLogger(__name__)
+
+ES_TIMEOUT = 60
+MONGO_TIMEOUT = 60
+
+
+@pytest.fixture(name="api_client")
+def fixture_api_client() -> APIClient:
+    """Create an API client for testing."""
+    return APIClient()
+
+
+def wait_for_mongodb() -> None:
+    """Wait for MongoDB to start."""
+    db = get_database()
+    timeout = ES_TIMEOUT
+    while timeout > 0:
+        try:
+            db.command("ping")
+            log.info("Connected to the MongoDB")
+            return
+        except ServerSelectionTimeoutError:
+            log.info("Waiting for mongodb to connect")
+            time.sleep(5)
+            timeout -= 5
+    raise Exception("Elasticsearch did not start in time")
```

Comment:
We can add this constant too:
```
SLEEP_INTERVAL = 5
```
nit: raise Exception("MongoDB did not start in time")